.github/workflows/release.yml

```diff
@@ -222,6 +222,27 @@ jobs:
     needs: [ build, version-tag ]
     if: needs.version-tag.outputs.append_notes == 'true'
```

Comment:
- The 'Get Release' step uses GitHub Actions 'github-script@v7', which is a good choice for interacting with the GitHub API.